packages/api/src/external/commonwell/link/shared.ts

```diff
@@ -161,6 +167,25 @@ export async function autoUpgradeNetworkLinks(
       }
     });
     await Promise.allSettled(upgradeRequests);
+  }
+}
+
+export async function getPatientsNetworkLinks(
+  commonWell: CommonWellAPI,
+  queryMeta: RequestMetadata,
+  commonwellPatientId: string
+): Promise<NetworkLink[]> {
+  const { debug, log } = out("CW getPatientNetworkLinks");
+
+  try {
+    const networkLinks = await commonWell.getNetworkLinks(queryMeta, commonwellPatientId);
+
+    debug(`resp getNetworkLinks: `, JSON.stringify(networkLinks));
+
+    if (!networkLinks._embedded.networkLink) {
```

Comment:
shouldn't we check that _embedded is truthy before using the networkLink prop, like we were doing previously?

Reply:
Since we're checking if its false in this case i updated it to !networkLinks._embedded? <- .networkLink. To make sure networkLinks._embedded isnt undefined